kaos/api/visibility.py

```diff
@@ -0,0 +1,105 @@
+"""This file contains routing code for the APIs. It should not contain much.
+
+Author: Team KMC-70.
+"""
+
+import json
+
+from flask import Blueprint, request, jsonify
+
+from .schema import SEARCH_QUERY_VALIDATOR
+from .validators import validate_request_schema
+from .errors import InputError
+from ..errors import ViewConeError
+from ..utils.time_conversion import utc_to_unix
+from ..models import DB, Satellite, ResponseHistory
+from ..algorithm.interpolator import Interpolator
+from ..algorithm.coord_conversion import lla_to_eci, lla_to_ecef, ecef_to_eci
+from ..algorithm.view_cone import reduce_poi
+from ..algorithm.visibility_finder import VisibilityFinder
+from ..tuples import TimeInterval
+
+# pylint: disable=invalid-name
+visibility_bp = Blueprint('visibility', __name__, url_prefix='/visibility')
+# pylint: enable=invalid-name
+
+
+@visibility_bp.route('/search', methods=['POST'])
+@validate_request_schema(SEARCH_QUERY_VALIDATOR)
+def get_satellite_visibility():
+    """Get the number of satellites that have visibility to a site.
+
+    Requires:
+        The user supplies a query param, 'coords.'
```

Comment:
This seems to be out of date. The user doesn't need any query params anymore, since everything is POSTed.